README.md

```diff
@@ -91,7 +91,9 @@ bal run
 
 The `Dropbox` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/module-ballerinax-dropbox/tree/main/examples/), covering the following use cases:
 
-[//]: # (TODO: Add examples)
+1. [Create a Folder](https://github.com/ballerina-platform/module-ballerinax-dropbox/tree/main/examples/create-folder) - Create a folder in Dropbox.
+
+2. [List files and folders in a Folder](https://github.com/ballerina-platform/module-ballerinax-dropbox/tree/main/examples/list-files-and-folders) - List all the files and folders in a folder in Dropbox.
```

Comment:
@rahul1995 I propose the below titles and descriptions for the examples to be more descriptive.
```suggestion
1. [Create folder in Dropbox](https://github.com/ballerina-platform/module-ballerinax-dropbox/tree/main/examples/create-folder) - Create a new folder in your Dropbox account.

2. [List folder contents in Dropbox](https://github.com/ballerina-platform/module-ballerinax-dropbox/tree/main/examples/list-files-and-folders) - Retrieve and display all files and subfolders within a specific folder in your Dropbox account.
```